subscription-engine/kafka-ws/Dockerfile

```diff
@@ -46,4 +46,4 @@ CMD ["./bin/entryPoint.sh", "npm", "run", "kafka-ws"]
 
 HEALTHCHECK --start-period=5s --interval=30s --timeout=5s --retries=2 \
     CMD HC=$(wget -q -O - "http://localhost:9000/health") \
```

Comment:
I'm not sure, but I think the command can be simpler: wget -q -O - "http://localhost:9000/health" > /dev/null 2>&1

Reply:
Yes, I think so
I used to use ` HEALTHCHECK --start-period=2m --interval=30s --timeout=10s --retries=3 \
 CMD curl -f http://localhost:9000/health || exit 1` I think it's simpler too